### PR TITLE
Add `filter` option

### LINKED ISSUE #1: Implement /fi for filtering
https://technet.microsoft.com/en-us/library/bb491010.aspx

### PR DESCRIPTION
**Pull request by kevva**
Fixes #1.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -14,3 +14,17 @@ test(function (t) {
 		t.assert(typeof d.memUsage === 'number');
 	});
 });
+
+test(function (t) {
+	t.plan(5);
+
+	tasklist({filter: ['status ne running']}	, function (err, data) {
```
Comment (kevva) [inline test.js:21]: Whoops, managed to sneak in a tab or something here I think.

Comment (sindresorhus) [inline test.js:21]: Already fixed it.

Comment (kevva) [inline test.js:21]: :(((((, same time as me.

Comment (kevva) [inline test.js:21]: I see you added some other fixes too though.

Comment (sindresorhus) [inline test.js:21]: all good

**Pull request merged**